index.html

```diff
@@ -1,45 +1,451 @@
+                  <p>If <var>CS</var> contains a member named <code>advanced</code>,
+                    return a promise [=reject|rejected=] with a newly
+                    [=exception/created=] {{TypeError}}.</p>
+                </li>
+                <li>
+                  <p>If <var>CS</var> contains a member whose name specifies a
+                    constrainable property applicable to
+                    <a data-cite="screen-capture#dfn-display-surface">display surfaces</a>,
+                    and whose value in turn is a dictionary containing a member
+                    named either <code>min</code> or <code>exact</code>, return
+                    a promise [=reject|rejected=] with a newly
+                    [=exception/created=] {{TypeError}}.
+                  </p>
+                </li>
+                <li>
+                  <p>If <var>CS</var> contains a member whose name specifies a
+                    constrainable property applicable to
+                    <a data-cite="screen-capture#dfn-display-surface">display surfaces</a>,
+                    and whose value in turn is a dictionary containing a member
+                    named <code>max</code>, and that member's value in turn is
+                    less than the constrainable property's
+                    <a data-cite="screen-capture#dfn-floor-value">floor value</a>,
+                    then let <var>failedConstraint</var> be the name of the
+                    member, let <var>message</var> be either
+                    <code>undefined</code> or an informative human-readable
+                    message, and return a promise [=reject|rejected=] with a new
+                    <code>OverconstrainedError</code> created by calling
+                    <code>OverconstrainedError(<var>failedConstraint</var>,
+                    <var>message</var>)</code>.
+                </li>
+              </ol>
+            </li>
+            <li>
+              <p>Let <var>requestedMediaTypes</var> be the set of media
+                types in <var>constraints</var> with either a dictionary
+                value or a value of <code>true</code>.</p>
+            </li>
+            <li>
+              <p>If the <a>current settings object</a>'s [=relevant global object=]'s
+                [=associated `Document`=] is NOT [=Document/fully active=] or does NOT
+                <a data-cite="!HTML/#gains-focus">have focus</a>, return a promise
+                [=reject|rejected=] with a {{DOMException}} object whose {{DOMException/name}}
+                attribute has the value {{InvalidStateError}}.
+              </p>
+            </li>
+            <li>
+              <p>Let <var>p</var> be a new promise.</p>
+            </li>
+            <li>
+              <p>Run the following steps in parallel:</p>
+              <ol>
+                <li>
+                  <p>For each media type <var>T</var> in
+                    <var>requestedMediaTypes</var>,
```

Comment:
I'd drop the for-each given that you're only supporting video.

Reply:
Done.